doc/modules/tree.rst

```diff
@@ -500,8 +500,17 @@ Entropy:
 
     H(Q_m) = - \sum_k p_{mk} \log(p_{mk})
 
+HellingerDistance
+
+.. math::
+
+    H(X_m) = \sqrt{(\sqrt{\frac{N_{left_0}}{N_{parent_0}}}-\sqrt{\frac{N_{left_1}}{N_{parent_1}}})^2+(\sqrt{\frac{N_{right_0}}{N_{parent_0}}}-\sqrt{\frac{N_{right_1}}{N_{parent_1}}})^2 }
```

Comment:
For consistency regarding previous notations.
```suggestion
HellingerDistance:

.. math::

    H(X_m) = \sqrt{\left(\sqrt{\frac{N^{left}_0}{N^{parent}_0}}-\sqrt{\frac{N^{left}_1}{N^{parent}_1}}\right)^2+\left(\sqrt{\frac{N^{right}_0}{N^{parent}_0}}-\sqrt{\frac{N^{right}_1}{N^{parent}_1}}\right)^2 }
```



![Screenshot 2022-01-10 at 17-06-38 1 10 Decision Trees](https://user-images.githubusercontent.com/13029839/148798240-cf4ac9d6-9fef-4273-8e14-43520f36273c.png)